backend/src/types/index.ts

```diff
@@ -0,0 +1 @@
+export * from './context';
```

Comment:
why did you create an extra interface for the Context? Is there a special reason?